test.js

```diff
@@ -1,36 +1,40 @@
-/* eslint-disable no-console */
 import assert from 'node:assert/strict'
-import { exec } from 'node:child_process'
+import { exec as execCallback } from 'node:child_process'
+import process from 'node:process'
 import { test } from 'node:test'
+import { promisify } from 'node:util'
 
-console.log('Running tests...')
+const exec = promisify(execCallback)
 
 test('current tests', async (t) => {
-  await t.todo('check if deprecation warning is shown', (_t, done) => {
-    exec('npm i request && npm run dev current', { timeout: 60000 }, (_error, _stdout, stderr) => {
-      assert.ok(/has been deprecated/.test(stderr), 'Expected "has been deprecated" to be mentioned in deprecation warning.')
-      done()
-    })
-  })
-
   await t.test('check if no deprecation warning is shown', (_t, done) => {
-    exec('npm run dev current', (_error, _stdout, stderr) => {
+    exec('node ./dist/cli.mjs current', (_error, _stdout, stderr) => {
       assert.ok(!/has been deprecated/.test(stderr), 'Not expected "has been deprecated" to be mentioned in deprecation warning.')
       done()
     })
   })
 
+  if (!process.version.startsWith('v16')) {
+    // Skip this test on Node.js v16 because I can't get it to work
+    await t.test('check if deprecation warning is shown if deprecated package is installed', async (_t) => {
+      const { stderr } = await exec('npx pnpm i request && node ./dist/cli.mjs current', { timeout: 160000 })
+      assert.ok(/request has been deprecated/.test(stderr), 'Expected "has been deprecated" to be mentioned in deprecation warning.')
+      // Cleanup: Undo the installation
+      await exec('pnpm remove request')
+    })
+  }
+
   await t.test('check if node version is mentioned in output', (_t, done) => {
-    exec('npm run dev current', (_error, stdout, _stderr) => {
-      assert.ok(/node version/.test(stdout), 'Expected "node version" to be mentioned in output.')
+    exec('node ./dist/cli.mjs current', (_error, stdout, stderr) => {
+      assert.ok(/node version/.test(stdout) || /node version/.test(stderr), 'Expected "node version" to be mentioned in output.')
```

Comment:
This should be another test, about `node ./dist/cli.mjs node`.

Reply:
> This should be another test, about `node ./dist/cli.mjs node`.

You're right. I just fixed it :slightly_smiling_face: